### ISSUE TITLE
this extension not work well with git

### ISSUE THREAD

**Issue by apm29** [Kruemelkatze/vscode-dashboard#93]
Type: <b>Bug</b>

install the buildin ext git and ProjectDashboard, ths source code manager can not show on the side bar!

Extension version: 2.5.2
VS Code version: Code 1.73.1 (Universal) (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T02:08:38.961Z)
OS version: Darwin x64 19.6.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4670 CPU @ 3.40GHz (4 x 3400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|4, 4, 4|
|Memory (System)|8.00GB (0.50GB free)|
|Process Argv|-psn_0_5317906 --crash-reporter-id ed1d8d0c-9215-44a1-9dfb-f54ab5d1b936|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30577860
nodejswelcome1cf:30587006
fc301958:30595537
3biah626:30602489
gswce2:30605431
3d0df643:30608728
dbltrim-noruby:30604474
89544117:30607850

```

</details>

**Comment by apm29** [comment]
> Thanks for reporting. Do you mean the default Source Control Option in the side bar? ![image](https://user-images.githubusercontent.com/5564731/201875832-49965fd7-ba14-411b-8e22-a04491738500.png)

yes, i can see the badge and hint on the project dashboard icon which indicate the number of source code changes, it seems that the  default source control's icon and the project dashboard'icon are merged as one.
![image](https://user-images.githubusercontent.com/23237855/201877161-7d576b29-3b27-46ee-baeb-abfe645f3cc5.png)

**Comment by Kruemelkatze** [comment]
What happens if you disable the dashboard extension?

From a first glance, I **think** that this out of the scope of this extension. For the icon, the extension only registers a view in VSCode and does not interfere with it's placement or rendering at all.

Unfortunately I'm unable to test it myself, as neither I nor any of my colleagues does have a Mac. :/

**Comment by apm29** [comment]
> What happens if you disable the dashboard extension?
> 
> From a first glance, I **think** that this out of the scope of this extension. For the icon, the extension only registers a view in VSCode and does not interfere with it's placement or rendering at all.
> 
> Unfortunately I'm unable to test it myself, as neither I nor any of my colleagues does have a Mac. :/

it seem ok when I disable the project dashboard extension. I'll try on my windows pc later.

**Comment by Kruemelkatze** [comment]
Thanks :)

**Comment by Kruemelkatze** [comment]
Any news on that?

**Comment by vboufleur** [comment]
@Kruemelkatze I'm having the same error, currently running vscode on Fedora Linux.

If the dashboard extension is enabled I can't open the "Source Control Option" on the sidebar, the Dashboard from this extension is opened instead. When the extension is disabled I can open it again.

My vscode version:
```bash
❯ code --version
1.95.2
e8653663e8840adaf45af01eab5c627a5af81807
x64
```

Can you please share some guiding tips on how to debug the issue locally and bring you more info that can help us find the root? Or maybe you require some info to reproduce this issue?

**Comment by vboufleur** [comment]
For anyone interested in this issue, I've deleted all `dashboard` options in my user preference file, uninstalled the extension and started fresh with a new install, and this issue is still happening. Every time I click on the "Source Control" tab it opens this extension instead.
